plugins/vocabbuilder.koplugin/db.lua

```diff
@@ -3,6 +3,7 @@ local Device = require("device")
 local SQ3 = require("lua-ljsqlite3/init")
 local LuaData = require("luadata")
 local logger = require("logger")
+local settings = G_reader_settings:readSetting("vocabulary_builder", {enabled = false, with_context = true})
```

Comment:
It... depends on what the setting is and how it is written.

If it's a table and the table itself is never completely rewritten, the reference should be stable, otherwise, a table rewrite with leave a dangling pointer (and a stale GC object).